sunset/code/__HELPERS/unique_ids.dm

```diff
@@ -0,0 +1,44 @@
+// Unique Datum Identifiers
```

Comment:
Da fuck is this?

Reply:
It is apart of faxes and if removed causes issues. However, once the other problems can be corrected, I'll see about removing this.